packages/teams-js/src/internal/telemetry.ts

```diff
@@ -1,4 +1,4 @@
-import { debug as registerLogger, Debugger } from 'debug';
+import { debug as registerLogger, Debugger } from 'debug/src/browser';
```

Comment:
>debug/src/browser [](http://example.com/codeflow?start=51&length=17)

Out of curiosity is this way of consuming the package "supported" by debug? Or are we kind of cheating?

Reply:
That kind of depends on what you mean by "supported." Do they have any official documentation pointing to this as a something they recommend? No. Though it's not like we're doing some crazy evil bit hacking here or anything. All we are doing is choosing our own entry point to bypass their index file. All the index file does is redirect either to the browser or node implementation since we are having an oddly specific bug that is causing problems with the implementation selection.